frontend/src/modules/search/components/SearchBox.js

```diff
@@ -3,6 +3,7 @@
 import * as React from 'react';
 import { connect } from 'react-redux';
 import debounce from 'lodash.debounce';
+import _ from 'lodash';
```

Comment:
We try not to import `lodash` directly, but instead install just the module that we need.

Reply:
Can I just import `_`  from `lodash-core`?

Reply:
No, please look at what we do just on the previous line. We want to import only the `mapValues` function, and nothing else. Just like, on the line before, we import only `debounce`. For that to work, you need to install `lodash.mapValues` in our frontend dependencies. That process is documented here: https://github.com/mozilla/pontoon/tree/master/frontend#dependencies

Reply:
I now see. Thank you for this.